.github/workflows/editorconfig.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'NixOS'
+    if: "github.repository_owner == 'NixOS' && !contains(github.event.pull_request.title, '[skip editorconfig]')"
```

Comment:
Can this be a label?  I think it would make PRs difficult to skim if it was part oy the title.

Reply:
Yes it could, but I think a label is easier to overlook or be mislabelled initially and adding the label _after_ the PR is opened don't avoid the API issue.

Reply:
I've updated this PR with the skip, I don't see that it is any worse.

Reply:
This will probably be used really little (mostly mass (test) formatting PRs) so I think it's fine and a label also wouldn't work as already said